src/main/resources/jenkins/branch/PropertyMigration/MonitorImpl/message.jelly

```diff
@@ -28,8 +28,9 @@
                 </div>
               </j:when>
               <j:otherwise>
+                <st:adjunct includes="jenkins.branch.PropertyMigration.MonitorImpl.branch-api-property-migration-disabled-form"/>
                 <form method="GET" action="${rootURL}/safeRestart" disabled="disabled"
-                      onsubmit="return false;"
+                      class="branch-api-property-migration-monitor-disabled-form"
                       style="float: none; display: inline; position: inherit;">
                   <span class="yui-button yui-link-button">
                     <span class="first-child">
```

Comment:
Alternatively we could take down this whole part that's inside the `otherwise` block. It's a no-op markup that renders an inactive form with an inactive button. And if someone re-enables the form and the button and submits the form the `return false;` statement will prevent the request to action URL.

I could speculate that this code is so rarely (if ever) reachable by now that nobody will ever notice the change here.